jupyter-notebooks/orders_basemaps/requests_order_basemaps.ipynb

```diff
@@ -0,0 +1,605 @@
+    "1. In the Basemap Viewer, enter your basemap ID, such as point_reyes_rolling_normalized_2020-04-30_mosaic\n",
+    "2. On the resulting basemap, select the arrow to view the quads and scenes, for example: 325-1260,326-1260,325-1259,326-1259\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Use Basemaps API to get the Basemap and quad IDs\n",
+    "---\n",
+    "\n",
+    "Use the Basemap APIs to retrieve the basemap you want and the quad IDs. You can get the base URL you'll need to communicate with the Basemap API service:\n",
+    "\n",
+    "1. Go to the [Basemap API reference](https://developers.planet.com/docs/basemaps/reference/) at https://developers.planet.com/docs/basemaps/reference/. \n",
+    "2. Under List Mosaics, select GET /mosaics.\n",
+    "\n",
+    "A dropdown UI element appears with the URL to use:\n",
+    "\n",
+    "https://api.planet.com/basemaps/v1/mosaics\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Search for the mosaic ID to get its quad IDs\n",
+    "\n",
+    "Here, we are using the full name of the mosaic, but with the `name__contains` field, you can use a partial name based on the names of the mosaics available through your account.\n",
+    "\n",
+    "Declare the search string for `requests.get()`. You will use the dictionary you create with the mosaic name to pass to the requests.get() function.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BASEMAP_API_URL = 'https://api.planet.com/basemaps/v1/mosaics'\n",
+    "headers = {'content-type': 'application/json'}\n",
+    "\n",
+    "basemap_params = {\n",
+    "    'name__contains': 'point_reyes_rolling_normalized_2020-04-30_mosaic'\n",
+    "}\n",
+    "\n",
+    "basemapServiceResponse = requests.get(\n",
+    "    url=BASEMAP_API_URL, params=basemap_params, auth=auth, headers=headers)\n",
+    "\n",
+    "basemaps = basemapServiceResponse.raise_for_status()\n",
+    "if basemapServiceResponse.status_code != 204:\n",
+    "    basemaps = json.loads(basemapServiceResponse.text)\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mosaicID = basemaps['mosaics'][0]['id']\n",
+    "mosaicBbox = basemaps['mosaics'][0]['bbox']\n",
+    "mosaicBboxStr = ','.join(map(str, mosaicBbox))\n",
+    "\n",
+    "quad_params = {\n",
+    "    'mosaic_id': mosaicID,\n",
+    "    'bbox': mosaicBboxStr,\n",
+    "}\n",
+    "\n",
+    "quads_url = \"{}/{}/quads\".format(BASEMAP_API_URL, mosaicID)\n",
+    "quadServiceResponse = requests.get(\n",
+    "    url=quads_url, params=quad_params, auth=auth, headers=headers)\n",
+    "quads = quadServiceResponse.json()\n",
+    "items = quads['items']\n",
+    "\n",
+    "quadIds = []\n",
+    "for i in items:\n",
```

Comment:
Perhaps a better way about going at it would be: `quad_id = [i['id'] for i in items]`